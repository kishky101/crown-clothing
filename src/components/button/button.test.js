import renderer from 'react-test-renderer';
import Button from './button.component';
//import { BaseButton, GoogleSignInButton, InvertedButton, ButtonSpinner } from './button.styles';

test('craeting button snapshot', () => {
    const mockprop = { 
        children: 'Go to Home',
        isLoading: false,
        buttonTypes: 'inverted'
    }
    const button = renderer.create(<Button prop = {mockprop} />).toJSON;
    expect(button).toMatchSnapshot();
})
