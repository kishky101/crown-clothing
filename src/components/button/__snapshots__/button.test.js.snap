// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`craeting button snapshot 1`] = `[Function]`;
